### ISSUE TITLE
Best Practices

### ISSUE THREAD

**Issue by alassiter** [freeCodeCamp/chapter#28]
Wanted to throw this in now. I think that once we land on some decisions re: frameworks, we should publish a best practices that this repo wants to follow. It will help people like me who have not contributed to open source yet jump on board and have some clear boundaries. Also, it can morph over time as we discuss changes.

**Comment by chrismgonzalez** [comment]
I like this idea - could be added from a link in the contributing guide.  (e.g. Best Practices/Style Guide.md)

**Comment by mquadrant** [comment]
Make sense.

**Comment by chrismgonzalez** [comment]
@alassiter I've added this to the thread about the DOCS - see #12

**Comment by nhsz** [comment]
Also adding formatter/linter like Prettier and StandardJS maybe? Or any other ESLint config

**Comment by chrismgonzalez** [comment]
I've added it to #12

**Comment by allella** [comment]
What's up @alassiter.  @chrismgonzalez  said he added this to #12 so how about we close this thread and post a summary or ant additional thoughts on that broader Docs thread?

**Comment by allella** [comment]
This is also a duplicate of #28 so @alassiter please close this one and post any other notes on #12.

Grazie

**Comment by allella** [comment]
Github also shares this link at the top of the main Issues page
https://opensource.guide/

**Comment by allella** [comment]
https://github.com/freeCodeCamp/chapter/blob/master/CONTRIBUTING.md should cover this and the docs in Pull Requests and @chrismgonzalez  outline should cover even more if it, so closing this one.

@alassiter what's up Greenville!
